src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/CopyAzureRmRecoveryServicesVault.cs

```diff
@@ -31,34 +31,48 @@ public class CopyAzureRmRecoveryServicesVault : RecoveryServicesBackupCmdletBase
     {
         #region Parameters
 
+        internal const string AzureRSVaultDataMoveParameterSet = "AzureRSVaultDataMoveParameterSet";
```

Comment:
Please sepecify a default parameter set for Copy-AzRecoveryServicesVault
```
  "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.RecoveryServices","Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.CopyAzureRmRecoveryServicesVault","Copy-AzRecoveryServicesVault","1","8510","Cmdlet 'Copy-AzRecoveryServicesVault' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Reply:
Specified and re-uploaded the PR